src/nidcpower/metadata/enums.py

```diff
@@ -957,27 +975,33 @@
     'SendSoftwareEdgeTriggerType': {
         'values': [
             {
-                'name': 'NIDCPOWER_VAL_START',
+                'name': 'NIDCPOWER_VAL_START_TRIGGER',
```

Comment:
Again, I think the changes here are OK since the new 'python_name' matches the old 'name' but not 100% sure.